apps/examples/testcase/le_tc/kernel/tc_mqueue.c

```diff
@@ -194,6 +194,52 @@ static int timedsend_test(void)
 		g_timedsend_mqfd = NULL;
 	}
 
+	/* Perform the timesend_test for NULL message queue parameter check & read permission cases */
+
+	memcpy(msg_buffer, TEST_MESSAGE, TEST_MSGLEN);
+
+	g_timedsend_mqfd = mq_open("t_mqueue", O_RDONLY, 0666, &attr);
+	if (g_timedsend_mqfd == (mqd_t)-1) {
+		printf("tc_mqueue_mq_timedsend_timedreceive FAIL: mq_open\n");
+		return ERROR;
+	}
+
+	/* Perform the send for above failure check cases */
+
+	status = mq_timedsend(g_timedsend_mqfd, msg_buffer, TEST_MSGLEN, 42, &st_time);
+	TC_ASSERT_EQ_RETURN("timedsend_test", status, ERROR, ERROR);
```

Comment:
This function is executed by another tc function. If you use TC_ASSERT in here, failure will be counted twice by tc_mqueue_mq_timedsend_timedreceive. Let's check it and return ERROR.